<!doctype html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-alpha.70">
<link rel="alternate" type="application/rss+xml" href="/blog/rss.xml" title="Data2Services Blog RSS Feed">
<link rel="alternate" type="application/atom+xml" href="/blog/atom.xml" title="Data2Services Blog Atom Feed">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:200,300,400,400i,500,600,700">
<link rel="stylesheet" href="/css/code-blocks-buttons.css">
<script src="https://buttons.github.io/buttons.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.0/clipboard.min.js"></script>
<script src="/js/code-blocks-buttons.js"></script><title data-react-helmet="true">Translator Prototypes | Data2Services</title><meta data-react-helmet="true" property="og:image" content="https://d2s.semanticscience.org//img/undraw_online.svg"><meta data-react-helmet="true" name="twitter:image" content="https://d2s.semanticscience.org//img/undraw_online.svg"><meta data-react-helmet="true" name="twitter:image:alt" content="Image for Data2Services"><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" name="docusaurus_locale" content="en"><meta data-react-helmet="true" name="docusaurus_version" content="current"><meta data-react-helmet="true" name="docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Translator Prototypes | Data2Services"><meta data-react-helmet="true" name="description" content="NCATS Biomedical Data Translator prototypes registry and documentation."><meta data-react-helmet="true" property="og:description" content="NCATS Biomedical Data Translator prototypes registry and documentation."><meta data-react-helmet="true" property="og:url" content="https://d2s.semanticscience.org//docs/translator-prototypes-registry"><link data-react-helmet="true" rel="shortcut icon" href="/img/d2s-logo.png"><link data-react-helmet="true" rel="canonical" href="https://d2s.semanticscience.org//docs/translator-prototypes-registry"><link rel="stylesheet" href="/styles.dbf913f8.css">
<link rel="preload" href="/styles.903fb0a7.js" as="script">
<link rel="preload" href="/runtime~main.7c9aaa28.js" as="script">
<link rel="preload" href="/main.c3417a16.js" as="script">
<link rel="preload" href="/1.7b322362.js" as="script">
<link rel="preload" href="/2.ab994efe.js" as="script">
<link rel="preload" href="/1f391b9e.64b390a9.js" as="script">
<link rel="preload" href="/70.e1de10fa.js" as="script">
<link rel="preload" href="/935f2afb.8c78b57b.js" as="script">
<link rel="preload" href="/17896441.2fd276c4.js" as="script">
<link rel="preload" href="/6a8fabba.e3f4939a.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<nav aria-label="Skip navigation links"><button type="button" tabindex="0" class="skipToContent_11B0">Skip to main content</button></nav><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><div aria-label="Navigation bar toggle" class="navbar__toggle" role="button" tabindex="0"><svg aria-label="Menu" width="30" height="30" viewBox="0 0 30 30" role="img" focusable="false"><title>Menu</title><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></div><a class="navbar__brand" href="/"><img src="/img/d2s-logo.png" alt="Data2Services" class="themedImage_YANc themedImage--light_3CMI navbar__logo"><img src="/img/d2s-logo.png" alt="Data2Services" class="themedImage_YANc themedImage--dark_3ARp navbar__logo"><strong class="navbar__title">Data2Services</strong></a><a aria-current="page" class="navbar__item navbar__link navbar__link--active" href="/docs/">Documentation</a><a class="navbar__item navbar__link" href="/blog/">Blog</a><a class="navbar__item navbar__link" href="/help">Help</a><a class="navbar__item navbar__link" href="/contributing">Contribute</a></div><div class="navbar__items navbar__items--right"><a href="https://github.com/MaastrichtU-IDS/d2s-docs" target="_blank" rel="noopener noreferrer" class="navbar__item navbar__link header-github-link"></a><div class="react-toggle react-toggle--disabled displayOnlyInLargeViewport_2N3Q"><div class="react-toggle-track"><div class="react-toggle-track-check"><span class="toggle_3NWk">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_3NWk">ðŸŒž</span></div></div><div class="react-toggle-thumb"></div><input type="checkbox" disabled="" aria-label="Dark mode toggle" class="react-toggle-screenreader-only"></div><div class="navbar__search searchBarContainer_1-U_"><input placeholder="Search" aria-label="Search" class="navbar__search-input"><div class="loadingRing_18Th searchBarLoadingRing_zYFK"><div></div><div></div><div></div><div></div></div></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div><div class="navbar-sidebar"><div class="navbar-sidebar__brand"><a class="navbar__brand" href="/"><img src="/img/d2s-logo.png" alt="Data2Services" class="themedImage_YANc themedImage--light_3CMI navbar__logo"><img src="/img/d2s-logo.png" alt="Data2Services" class="themedImage_YANc themedImage--dark_3ARp navbar__logo"><strong class="navbar__title">Data2Services</strong></a></div><div class="navbar-sidebar__items"><div class="menu"><ul class="menu__list"><li class="menu__list-item"><a aria-current="page" class="menu__link navbar__link--active" href="/docs/">Documentation</a></li><li class="menu__list-item"><a class="menu__link" href="/blog/">Blog</a></li><li class="menu__list-item"><a class="menu__link" href="/help">Help</a></li><li class="menu__list-item"><a class="menu__link" href="/contributing">Contribute</a></li><li class="menu__list-item"><a href="https://github.com/MaastrichtU-IDS/d2s-docs" target="_blank" rel="noopener noreferrer" class="menu__link header-github-link"></a></li></ul></div></div></div></nav><div class="main-wrapper"><div class="docPage_vMrn"><div class="docSidebarContainer_3Ak5" role="complementary"><div class="sidebar_3gvy"><div class="menu menu--responsive thin-scrollbar menu_1yIk"><button aria-label="Open Menu" aria-haspopup="true" class="button button--secondary button--sm menu__button" type="button"><svg aria-label="Menu" class="sidebarMenuIcon_1CUI" width="24" height="24" viewBox="0 0 30 30" role="img" focusable="false"><title>Menu</title><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><ul class="menu__list"><li class="menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#!">Publish data</a><ul class="menu__list"><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/">Introduction</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/installation">Installation</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/data-model">Define the data model</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/data-validation">RDF data validation</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/use-python">Convert with Python</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/convert-rml">Convert with RML</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/store-rdf">Store RDF data</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/sparql-queries">Write SPARQL queries</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/generate-metadata">Generate metadata</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/publish-fair-data">Publish FAIR data</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/workflow-github">Deploy workflows</a></li></ul></li><li class="menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#!">Deploy services</a><ul class="menu__list"><li class="menu__list-item"><a class="menu__link" tabindex="0" href="/docs/deploy-services">Recommended services</a></li><li class="menu__list-item"><a class="menu__link" tabindex="0" href="/docs/deploy-additional-services">Additional services</a></li><li class="menu__list-item"><a aria-current="page" class="menu__link menu__link--active active" tabindex="0" href="/docs/translator-prototypes-registry">Translator Prototypes</a></li></ul></li><li class="menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#!">Guides</a><ul class="menu__list"><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-preprocessing">Preprocess input files</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-docker">Docker guide</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-jenkins">Experimental Jenkins workflows</a></li><li class="menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#!" tabindex="-1">Setup databases</a><ul class="menu__list"><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-graphdb">Setting up GraphDB</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-virtuoso">Setting up Virtuoso</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-postgres">Setting up PostgreSQL</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-mariadb">Setting up MariaDB</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/guide-mysql">Setting up MySQL</a></li></ul></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/translator-standards">Translator Standards</a></li></ul></li><li class="menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#!">Kubernetes workflows</a><ul class="menu__list"><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/argo-install">Argo installation</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/argo-login">Login to OpenShift</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/argo-volume">Define volumes</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/argo-run">Run Argo workflows</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/argo-commands">Argo commands</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/argo-debug">Debug Argo workflow</a></li><li class="menu__list-item"><a class="menu__link" tabindex="-1" href="/docs/argo-secret">Use OpenShift secrets</a></li></ul></li></ul></div><button type="button" title="Collapse sidebar" aria-label="Collapse sidebar" class="button button--secondary button--outline collapseSidebarButton_2WRA"><svg width="20" height="20" role="img" class="collapseSidebarButtonIcon_CoMu"><g fill="#7a7a7a"><path d="M9.992 10.023c0 .2-.062.399-.172.547l-4.996 7.492a.982.982 0 01-.828.454H1c-.55 0-1-.453-1-1 0-.2.059-.403.168-.551l4.629-6.942L.168 3.078A.939.939 0 010 2.528c0-.548.45-.997 1-.997h2.996c.352 0 .649.18.828.45L9.82 9.472c.11.148.172.347.172.55zm0 0"></path><path d="M19.98 10.023c0 .2-.058.399-.168.547l-4.996 7.492a.987.987 0 01-.828.454h-3c-.547 0-.996-.453-.996-1 0-.2.059-.403.168-.551l4.625-6.942-4.625-6.945a.939.939 0 01-.168-.55 1 1 0 01.996-.997h3c.348 0 .649.18.828.45l4.996 7.492c.11.148.168.347.168.55zm0 0"></path></g></svg></button></div></div><main class="docMainContainer_2iGs"><div class="container padding-vert--lg docItemWrapper_1bxp"><div class="row"><div class="col docItemCol_U38p"><div class="docItemContainer_a7m4"><article><header><h1 class="docTitle_Oumm">Translator Prototypes</h1></header><div class="markdown"><p><a href="https://ncats.nih.gov/translator" target="_blank" rel="noopener noreferrer">NCATS Biomedical Data Translator</a> prototypes registry and documentation.</p><blockquote><p>See the official Translator prototypes registry at <a href="https://ncatstranslator.github.io" target="_blank" rel="noopener noreferrer">https://ncatstranslator.github.io</a></p></blockquote><h2><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="standard-reasoner-implementations-sri"></a>Standard Reasoner Implementations (SRI)<a class="hash-link" href="#standard-reasoner-implementations-sri" title="Direct link to heading">#</a></h2><p>Services to explore and validate implementations against the Translator standards.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="reasonerapi"></a>ReasonerAPI<a class="hash-link" href="#reasonerapi" title="Direct link to heading">#</a></h3><p><a href="https://github.com/NCATSTranslator/ReasonerAPI" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/NCATSTranslator/ReasonerAPI?label=ReasonerAPI&amp;style=social" alt="ReasonerAPI"></a></p><p>Standard recommended when serving an API in the Translator project. It consists in a JSON model for graph data, and allow to query and get answer from Translator APIs.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="kgx"></a>KGX<a class="hash-link" href="#kgx" title="Direct link to heading">#</a></h3><p><a href="https://github.com/NCATS-Tangerine/kgx" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/NCATS-Tangerine/kgx?label=kgx&amp;style=social" alt="kgx"></a></p><p>KGX (Knowledge Graph Exchange) is a Python library and set of command  line utilities for exchanging Knowledge Graphs (KGs) that conform to or  are aligned to the <a href="https://biolink.github.io/biolink-model/" target="_blank" rel="noopener noreferrer">Biolink Model</a>.</p><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access"></a>Access<a class="hash-link" href="#access" title="Direct link to heading">#</a></h5><ul><li>See <a href="https://docs.google.com/presentation/d/1DHw-499n3mrjBnziSXYV_cCij4QMndrEH2IGWz8-Kzw/edit#slide=id.p" target="_blank" rel="noopener noreferrer">KGX presentation</a>.</li><li>See <a href="https://github.com/NCATS-Tangerine/kgx/blob/master/data-preparation.md" target="_blank" rel="noopener noreferrer">KGX data preparation documentation</a>.</li><li>See the <a href="https://github.com/NCATSTranslator/Knowledge-Graph-Exchange-Working-Group" target="_blank" rel="noopener noreferrer">GitHub repository of the Knowledge Graph Exchange Working Group</a></li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="nodenormalization-service"></a>NodeNormalization Service<a class="hash-link" href="#nodenormalization-service" title="Direct link to heading">#</a></h3><p><a href="https://github.com/TranslatorIIPrototypes/NodeNormalization" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/TranslatorIIPrototypes/NodeNormalization?label=NodeNormalization&amp;style=social" alt="NodeNormalization"></a></p><ul><li>Open API: <a href="https://nodenormalization-sri.renci.org/apidocs/" target="_blank" rel="noopener noreferrer">https://nodenormalization-sri.renci.org/apidocs/</a> </li><li><a href="https://github.com/TranslatorIIPrototypes/NodeNormalization/blob/master/documentation/NodeNormalization.ipynb" target="_blank" rel="noopener noreferrer">NodeNormalization Jupyter Notebook</a> for documentation</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="edgenormalization-service"></a>EdgeNormalization Service<a class="hash-link" href="#edgenormalization-service" title="Direct link to heading">#</a></h3><p><a href="https://github.com/TranslatorIIPrototypes/EdgeNormalization" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/TranslatorIIPrototypes/EdgeNormalization?label=EdgeNormalization&amp;style=social" alt="EdgeNormalization"></a></p><ul><li>Open API: <a href="https://edgenormalization-sri.renci.org/apidocs/" target="_blank" rel="noopener noreferrer">https://edgenormalization-sri.renci.org/apidocs/</a></li><li><a href="https://github.com/TranslatorIIPrototypes/EdgeNormalization/blob/master/documentation/EdgeNormalization.ipynb" target="_blank" rel="noopener noreferrer">EdgeNormnalization Jupyter Notebook</a> for documentation</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="nameresolution"></a>NameResolution<a class="hash-link" href="#nameresolution" title="Direct link to heading">#</a></h3><p><a href="https://github.com/TranslatorIIPrototypes/NameResolution" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/TranslatorIIPrototypes/NameResolution?label=NameResolution&amp;style=social" alt="NameResolution"></a></p><ul><li>Open API: <a href="http://robokop.renci.org:2434/docs" target="_blank" rel="noopener noreferrer">http://robokop.renci.org:2434/docs</a></li><li><a href="https://github.com/TranslatorIIPrototypes/NameResolution/blob/master/documentation/NameResolution.ipynb" target="_blank" rel="noopener noreferrer">NameResolution Jupyter Notebook</a> for documentation</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="biolink-model-lookup-service"></a>BioLink Model Lookup service<a class="hash-link" href="#biolink-model-lookup-service" title="Direct link to heading">#</a></h3><p><a href="https://github.com/TranslatorIIPrototypes/bl_lookup" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/TranslatorIIPrototypes/bl_lookup?label=BL_lookup&amp;style=social" alt="BioLink Lookup"></a></p><ul><li>Open API: <a href="https://bl-lookup-sri.renci.org/apidocs/" target="_blank" rel="noopener noreferrer">https://bl-lookup-sri.renci.org/apidocs/</a></li><li><a href="https://github.com/TranslatorIIPrototypes/bl_lookup/blob/master/documentation/BiolinkLookup.ipynb" target="_blank" rel="noopener noreferrer">BioLink Lookup Jupyter Notebook</a> for documentation</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="blcompliance"></a>BLCompliance<a class="hash-link" href="#blcompliance" title="Direct link to heading">#</a></h3><p><a href="https://github.com/TranslatorIIPrototypes/BLComplianceService" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/TranslatorIIPrototypes/BLComplianceService?label=BLComplianceService&amp;style=social" alt="BLCompliance"></a></p><ul><li>Open API: <a href="https://monarch-sandbox.cgrb.oregonstate.edu/docs" target="_blank" rel="noopener noreferrer">https://monarch-sandbox.cgrb.oregonstate.edu/docs</a></li><li><a href="https://github.com/TranslatorIIPrototypes/BLComplianceService/blob/master/documentation/BLCompliance.ipynb" target="_blank" rel="noopener noreferrer">BLCompliance Jupyter Notebook</a> for documentation</li></ul><h2><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="knowledge-providers-kp"></a>Knowledge Providers (KP)<a class="hash-link" href="#knowledge-providers-kp" title="Direct link to heading">#</a></h2><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="cohd-clinical-data2services-provider"></a>COHD Clinical Data2Services Provider<a class="hash-link" href="#cohd-clinical-data2services-provider" title="Direct link to heading">#</a></h3><p><a href="https://github.com/WengLab-InformaticsResearch/cohd_api" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/WengLab-InformaticsResearch/cohd_api?label=cohd-api&amp;style=social" alt="COHD"></a></p><ul><li>Columbia Open Health Data (COHD)<ul><li>Clinical associations mined from observational EHR data</li><li>Conditions, drugs, procedures, gender, race, ethnicity</li><li>EHR prevalence and Co-occurrence count</li><li>Associations calculated from EHR prevalence and co-occurrence count</li><li>Privacy protection measures</li></ul></li></ul><p><a href="https://github.com/MaastrichtU-IDS/d2s-docs" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/MaastrichtU-IDS/d2s-docs?label=d2s-docs&amp;style=social" alt="Data2Services documentation"></a></p><ul><li>Data2Services<ul><li>A framework and Command Line Interface for building and deploying Translator data and services in a reproducible manner.</li><li>Documentation and tools to transform your data to a BioLink-compliant RDF knowledge graph</li><li>Automatically deploy a Reasoner API over a <a href="https://biolink.github.io/biolink-model/docs/" target="_blank" rel="noopener noreferrer">BioLink-compliant</a> RDF triplestore</li><li>Deploy additional interfaces to explore the knowledge graph</li></ul></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-1"></a>Access<a class="hash-link" href="#access-1" title="Direct link to heading">#</a></h5><ul><li>COHD Reasoner API at <a href="http://cohd.io/api" target="_blank" rel="noopener noreferrer">http://cohd.io/api</a></li><li>Data2Services documentation at <a href="https://d2s.semanticscience.org" target="_blank" rel="noopener noreferrer">https://d2s.semanticscience.org</a></li><li>Data2Services Reasoner API over BioLink RDF at <a href="http://api.trek.semanticscience.org" target="_blank" rel="noopener noreferrer">http://api.trek.semanticscience.org</a> (see on <a href="https://github.com/MaastrichtU-IDS/d2s-api" target="_blank" rel="noopener noreferrer">GitHub</a>)</li><li>Into-the-graph web UI to browse a BioLink RDF triplestore leveraging metadata and services at <a href="http://trek.semanticscience.org" target="_blank" rel="noopener noreferrer">http://trek.semanticscience.org</a></li><li>GitHub repositories for Data2Services <a href="https://github.com/MaastrichtU-IDS/d2s-project-template" target="_blank" rel="noopener noreferrer">project template</a> and <a href="https://github.com/MaastrichtU-IDS/d2s-cli" target="_blank" rel="noopener noreferrer">command line interface</a>.</li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="deploy"></a>Deploy<a class="hash-link" href="#deploy" title="Direct link to heading">#</a></h5><div class="mdxCodeBlock_1zKU"><div class="codeBlockContent_actS"><div tabindex="0" class="prism-code language-shell codeBlock_tuNs thin-scrollbar"><div class="codeBlockLines_3uvA" style="color:#bfc7d5;background-color:#292d3e"><div class="token-line" style="color:#bfc7d5"><span class="token plain">pip </span><span class="token function" style="color:rgb(130, 170, 255)">install</span><span class="token plain"> d2s cwlref-runner</span></div><div class="token-line" style="color:#bfc7d5"><span class="token plain">d2s init my-project</span></div></div></div><button type="button" aria-label="Copy code to clipboard" class="copyButton_2GIj">Copy</button></div></div><blockquote><p>Docker must be installed.</p></blockquote><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="molepro-molecular-data-provider"></a>MolePro Molecular Data Provider<a class="hash-link" href="#molepro-molecular-data-provider" title="Direct link to heading">#</a></h3><p><a href="https://github.com/broadinstitute/molecular-data-provider" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/broadinstitute/molecular-data-provider?label=molecular-data-provider&amp;style=social" alt="MolePro"></a></p><p>A Molecular Data Provider translating molecular scale to systems scale through a Reasoner API.</p><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-2"></a>Access<a class="hash-link" href="#access-2" title="Direct link to heading">#</a></h5><ul><li>Open API: <a href="https://translator.broadinstitute.org/molecular_data_provider/api" target="_blank" rel="noopener noreferrer">https://translator.broadinstitute.org/molecular_data_provider/api</a></li><li>Reasoner API: <a href="https://smart-api.info/ui/912372f46127b79fb387cd2397203709#" target="_blank" rel="noopener noreferrer">https://smart-api.info/ui/912372f46127b79fb387cd2397203709#</a></li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="genetic-knowledge-provider"></a>Genetic Knowledge Provider<a class="hash-link" href="#genetic-knowledge-provider" title="Direct link to heading">#</a></h3><p><a href="https://github.com/broadinstitute/genetics-kp-dev" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/broadinstitute/genetics-kp-dev?label=genetics-kp-dev&amp;style=social" alt="Genetic Knowledge Provider"></a></p><p>A tool to curate genetic associations for complex diseases, interpret their biological effects, and make these data available to the Translator.</p><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-3"></a>Access<a class="hash-link" href="#access-3" title="Direct link to heading">#</a></h5><ul><li>Reasoner API: <a href="https://translator.broadinstitute.org/genetics_data_provider/query" target="_blank" rel="noopener noreferrer">https://translator.broadinstitute.org/genetics_data_provider/query</a></li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="icees-kp-exposure-provider"></a>ICEES+ KP Exposure Provider<a class="hash-link" href="#icees-kp-exposure-provider" title="Direct link to heading">#</a></h3><p><a href="https://github.com/NCATS-Tangerine/icees-api" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/NCATS-Tangerine/icees-api?label=icees-api&amp;style=social" alt="ICEES+ KP Exposure Provider"></a></p><ul><li>Patient data + environmental exposures data</li><li>Integrated at patient- and visit-level</li><li>UNC Health Care System (UNCHCS) + NIEHS Environmental Polymorphisms Registry (EPR)</li><li>Observational EHR data, EPR survey data, SNP data, exposures data </li><li>Available for years 2010 â€“ 2016</li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-4"></a>Access<a class="hash-link" href="#access-4" title="Direct link to heading">#</a></h5><ul><li>ICEES+ Open APIs<ul><li>API KP: <a href="http://icees.renci.org:16339/apidocs" target="_blank" rel="noopener noreferrer">http://icees.renci.org:16339/apidocs</a></li><li>API UNC: <a href="https://icees.renci.org:16335/apidocs" target="_blank" rel="noopener noreferrer">https://icees.renci.org:16335/apidocs</a></li><li>API NIEHS: <a href="https://icees.renci.org:16336/apidocs" target="_blank" rel="noopener noreferrer">https://icees.renci.org:16336/apidocs</a></li><li>API Duke: <a href="https://icees.renci.org:16337/apidocs" target="_blank" rel="noopener noreferrer">https://icees.renci.org:16337/apidocs</a></li><li>Prototype ICEES+ Wireframe UI: <a href="http://robokop.renci.org:3001" target="_blank" rel="noopener noreferrer">http://robokop.renci.org:3001</a></li></ul></li><li>TranQL: <ul><li>Web UI: <a href="https://tranql.renci.org" target="_blank" rel="noopener noreferrer">https://tranql.renci.org</a></li><li><a href="https://github.com/NCATS-Tangerine/tranql/blob/master/tranql/queries/ICEES+EPR_sample_query" target="_blank" rel="noopener noreferrer">Sample TranQL queries</a></li></ul></li><li>Components<ul><li>FHIR-PIT: <a href="https://github.com/NCATS-Tangerine/FHIR-PIT" target="_blank" rel="noopener noreferrer">https://github.com/NCATS-Tangerine/FHIR-PIT</a></li><li>Secure Multiparty Computation: <a href="https://github.com/RENCI-NRIG/impact-smc" target="_blank" rel="noopener noreferrer">https://github.com/RENCI-NRIG/impact-smc</a></li><li>Machine learning: <a href="https://github.com/NCATS-Tangerine/iceesnn" target="_blank" rel="noopener noreferrer">https://github.com/NCATS-Tangerine/iceesnn</a></li></ul></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="deploy-1"></a>Deploy<a class="hash-link" href="#deploy-1" title="Direct link to heading">#</a></h5><div class="mdxCodeBlock_1zKU"><div class="codeBlockContent_actS"><div tabindex="0" class="prism-code language-shell codeBlock_tuNs thin-scrollbar"><div class="codeBlockLines_3uvA" style="color:#bfc7d5;background-color:#292d3e"><div class="token-line" style="color:#bfc7d5"><span class="token function" style="color:rgb(130, 170, 255)">git</span><span class="token plain"> clone https://github.com/NCATS-Tangerine/icees-api.git</span></div><div class="token-line" style="color:#bfc7d5"><span class="token plain"></span><span class="token builtin class-name" style="color:rgb(255, 203, 107)">cd</span><span class="token plain"> icees-api</span></div><div class="token-line" style="color:#bfc7d5"><span class="token plain"></span><span class="token comment" style="color:rgb(105, 112, 152);font-style:italic"># Edit .env</span><span class="token plain"></span></div><div class="token-line" style="color:#bfc7d5"><span class="token plain"></span><span class="token function" style="color:rgb(130, 170, 255)">docker-compose</span><span class="token plain"> up --build</span></div></div></div><button type="button" aria-label="Copy code to clipboard" class="copyButton_2GIj">Copy</button></div></div><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="text-mining-provider"></a>Text Mining Provider<a class="hash-link" href="#text-mining-provider" title="Direct link to heading">#</a></h3><p><a href="https://github.com/NCATSTranslator/Text-Mining-Provider-Roadmap" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/NCATSTranslator/Text-Mining-Provider-Roadmap?label=Text-Mining-Provider-Roadmap&amp;style=social" alt="Text Mining Provider roadmap"></a></p><p>Up-to-date, BioLink-compatible, knowledge graph composed of assertions mined from the available full-text biomedical literature using high-performance text mining systems</p><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="connections-hypothesis-provider"></a>Connections Hypothesis Provider<a class="hash-link" href="#connections-hypothesis-provider" title="Direct link to heading">#</a></h3><ul><li>Access Heterogeneous Data<ul><li>Researcher clinical data</li><li>Knowledge captured in the Biomedical Data Translator project</li></ul></li><li>Automate Source Selection</li><li>Effective Question-Response Ranking</li><li>Actionable Information</li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="docket-multiomics-provider"></a>DOCKET multiomics provider<a class="hash-link" href="#docket-multiomics-provider" title="Direct link to heading">#</a></h3><p><a href="https://github.com/PriceLab/DOCKET" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/PriceLab/DOCKET?label=DOCKET&amp;style=social" alt="DOCKET multiomics provider"></a></p><ul><li><p>Big GIM (Gene Interaction Miner), function interaction data for all pairs of genes.  Functional interaction data are available from four different sources:  1) tissue-specific gene expression correlations from healthy tissue  samples (GTEx), 2) tissue-specific gene expression correlations from  cancer samples (TCGA), 3) tissue-specific probabilities of function  interaction (GIANT), and 4) direct interactions (BioGRID).</p></li><li><p>Big CLAM (Cell Line Association Miner), integrates large-scale high-quality data of various cell line  resources to uncover associations between genomic and molecular features of cell lines, drug response measurements and gene knockdown viability  scores. The cell line data comes from five different sources: 1) CCLE -  Cancer Cell Line Encyclopedia, 2) GDSC - Genomics of Drug Sensitivity in Cancer, 3) CTRP - Cancer Therapeutics Response Portal, 4) CMap -  Connectivity Map, and 5) CDM - Cancer Dependency Map.</p></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-5"></a>Access<a class="hash-link" href="#access-5" title="Direct link to heading">#</a></h5><ul><li>Reasoner API: <a href="http://biggim.ncats.io/api" target="_blank" rel="noopener noreferrer">http://biggim.ncats.io/api</a></li><li><a href="https://drive.google.com/drive/u/0/folders/19CT2bu1kzVnXgORhgIQijJd7x8O8Ez6D" target="_blank" rel="noopener noreferrer">Running instructions</a></li><li>Big GIM II API: <a href="https://github.com/gloriachin/BigGIMII_API" target="_blank" rel="noopener noreferrer">https://github.com/gloriachin/BigGIMII_API</a></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="deploy-2"></a>Deploy<a class="hash-link" href="#deploy-2" title="Direct link to heading">#</a></h5><p>Documentation and integration to <code>d2s</code> <a href="https://d2s.semanticscience.org/docs/services-utilities#docket-multiomics-data-provider" target="_blank" rel="noopener noreferrer">started here</a>.</p><div class="mdxCodeBlock_1zKU"><div class="codeBlockContent_actS"><div tabindex="0" class="prism-code language-shell codeBlock_tuNs thin-scrollbar"><div class="codeBlockLines_3uvA" style="color:#bfc7d5;background-color:#292d3e"><div class="token-line" style="color:#bfc7d5"><span class="token plain">d2s start docket</span></div></div></div><button type="button" aria-label="Copy code to clipboard" class="copyButton_2GIj">Copy</button></div></div><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="biothings-api"></a>BioThings API<a class="hash-link" href="#biothings-api" title="Direct link to heading">#</a></h3><p><a href="https://github.com/biothings/biothings.api" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/biothings/biothings.api?label=biothings.api&amp;style=social" alt="BioThings API"></a></p><p>Build and deploy BioThings APIs from flat data files.</p><ul><li>API-fy knowledge sources on demand</li><li>Use BioThings SDK in Python to download and parse input data sources</li><li>Integrate your API to a meta-KG using Smart API</li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-6"></a>Access<a class="hash-link" href="#access-6" title="Direct link to heading">#</a></h5><ul><li><p>Translator KP APIs powered by BioThings SDK: <a href="https://biothings.ncats.io" target="_blank" rel="noopener noreferrer">https://biothings.ncats.io</a></p></li><li><p>BioThings SDK</p><ul><li>Docs: <a href="https://docs.biothings.io" target="_blank" rel="noopener noreferrer">https://docs.biothings.io</a></li><li>PyPI package: <a href="https://pypi.org/project/biothings" target="_blank" rel="noopener noreferrer">https://pypi.org/project/biothings</a></li></ul></li><li><p>SmartAPI: <a href="https://smart-api.info" target="_blank" rel="noopener noreferrer">https://smart-api.info</a></p><ul><li>GitHub: <a href="https://github.com/SmartAPI/smartAPI" target="_blank" rel="noopener noreferrer">https://github.com/SmartAPI/smartAPI</a></li><li>Up-to-date Meta-KG: <a href="https://smart-api.info/registry/translator/meta-kg" target="_blank" rel="noopener noreferrer">https://smart-api.info/registry/translator/meta-kg</a> </li></ul></li></ul><ul><li><p>Example Disease KP API: <a href="https://biothings.ncats.io/DISEASES" target="_blank" rel="noopener noreferrer">https://biothings.ncats.io/DISEASES</a></p><ul><li>Knowledge source: <a href="https://diseases.jensenlab.org" target="_blank" rel="noopener noreferrer">https://diseases.jensenlab.org</a></li><li>GitHub: <a href="https://github.com/kevinxin90/DISEASES" target="_blank" rel="noopener noreferrer">https://github.com/kevinxin90/DISEASES</a></li></ul></li><li><p>Up-to-date Meta-KG: <a href="https://smart-api.info/registry/translator/meta-kg" target="_blank" rel="noopener noreferrer">https://smart-api.info/registry/translator/meta-kg</a> </p></li><li><p>Service KP milestone dashboard: <a href="https://github.com/orgs/biothings/projects/5" target="_blank" rel="noopener noreferrer">https://github.com/orgs/biothings/projects/5</a></p></li><li><p>BioThings Studio</p><ul><li>GitHub: <a href="https://github.com/biothings/biothings_studio" target="_blank" rel="noopener noreferrer">https://github.com/biothings/biothings_studio</a></li><li>Docs: <a href="https://docs.biothings.io/en/latest/doc/studio.html" target="_blank" rel="noopener noreferrer">https://docs.biothings.io/en/latest/doc/studio.html</a></li></ul></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="deploy-3"></a>Deploy<a class="hash-link" href="#deploy-3" title="Direct link to heading">#</a></h5><p>Documentation and integration to <code>d2s</code> <a href="https://d2s.semanticscience.org/docs/d2s-biothings" target="_blank" rel="noopener noreferrer">started here</a>. See the <a href="https://docs.biothings.io/en/latest/doc/studio.html" target="_blank" rel="noopener noreferrer">BioLink Studio documentation</a>.</p><div class="mdxCodeBlock_1zKU"><div class="codeBlockContent_actS"><div tabindex="0" class="prism-code language-shell codeBlock_tuNs thin-scrollbar"><div class="codeBlockLines_3uvA" style="color:#bfc7d5;background-color:#292d3e"><div class="token-line" style="color:#bfc7d5"><span class="token plain">d2s start biothings-studio</span></div></div></div><button type="button" aria-label="Copy code to clipboard" class="copyButton_2GIj">Copy</button></div></div><hr><h2><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="autonomous-relay-agent-ara"></a>Autonomous Relay Agent (ARA)<a class="hash-link" href="#autonomous-relay-agent-ara" title="Direct link to heading">#</a></h2><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="biothings-explorer"></a>BioThings Explorer<a class="hash-link" href="#biothings-explorer" title="Direct link to heading">#</a></h3><p><a href="https://github.com/biothings/biothings_explorer" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/biothings/biothings_explorer?label=biothings-explorer&amp;style=social" alt="BioThings Explorer"></a></p><p>Federated querying of BioThings APIs, done in 2 steps:</p><ul><li>Build a query path plan defining APIs relevant to answer the query</li><li>Execute the query path plan to retrieve data from the different APIs.</li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-7"></a>Access<a class="hash-link" href="#access-7" title="Direct link to heading">#</a></h5><ul><li>BioThings Explorer UI demo: <a href="https://biothings.io/explorer" target="_blank" rel="noopener noreferrer">https://biothings.io/explorer</a></li><li>Docs: <a href="https://biothings_explorer.readthedocs.io/en/latest" target="_blank" rel="noopener noreferrer">https://biothings_explorer.readthedocs.io/en/latest</a></li><li><a href="https://github.com/biothings/biothings_explorer/tree/master/jupyter%20notebooks" target="_blank" rel="noopener noreferrer">Jupyter Notebooks on gitHub</a></li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="aragorn"></a>ARAGORN<a class="hash-link" href="#aragorn" title="Direct link to heading">#</a></h3><p><strong>Autonomous Relay Agent for Generation Of Ranked Networks</strong>. A tool to query Knowledge Providers (KPs) and synthesize highly ranked answers relevant to user-specified questions</p><ul><li>operate in a federated knowledge environment</li><li>bridge the precision mismatch between data specificity in KPs and more abstract level of user queries</li><li>generalize answer ranking</li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="tools"></a>Tools<a class="hash-link" href="#tools" title="Direct link to heading">#</a></h5><p><a href="https://github.com/ranking-agent/QuestionRewrite" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/ranking-agent/QuestionRewrite?label=QuestionRewrite&amp;style=social" alt="QuestionRewrite"></a></p><ul><li>Question Augmentation<ul><li>Open API: <a href="https://questionaugmentation.renci.org/apidocs" target="_blank" rel="noopener noreferrer">https://questionaugmentation.renci.org/apidocs</a></li><li>Example Notebooks:<ul><li><a href="https://github.com/ranking-agent/QuestionRewrite/blob/master/documentation/QuestionAugmentationSimilarity_strider.ipynb" target="_blank" rel="noopener noreferrer">https://github.com/ranking-agent/QuestionRewrite/blob/master/documentation/QuestionAugmentationSimilarity_strider.ipynb</a></li><li><a href="https://github.com/ranking-agent/QuestionRewrite/blob/master/documentation/QuestionAugmentationEdges.ipynb" target="_blank" rel="noopener noreferrer">https://github.com/ranking-agent/QuestionRewrite/blob/master/documentation/QuestionAugmentationEdges.ipynb</a></li></ul></li></ul></li></ul><p><a href="https://github.com/ranking-agent/AnswerCoalesce" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/ranking-agent/AnswerCoalesce?label=AnswerCoalesce&amp;style=social" alt="AnswerCoalesce"></a></p><ul><li>Answer Coalescence<ul><li>Open API: <a href="https://answercoalesce.renci.org/apidocs/" target="_blank" rel="noopener noreferrer">https://answercoalesce.renci.org/apidocs/</a></li><li>Example Notebook: <a href="https://github.com/ranking-agent/AnswerCoalesce/blob/master/documentation/AnswerCoalescence.ipynb" target="_blank" rel="noopener noreferrer">https://github.com/ranking-agent/AnswerCoalesce/blob/master/documentation/AnswerCoalescence.ipynb</a></li></ul></li></ul><p><a href="https://github.com/ranking-agent/gamma-viewer" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/ranking-agent/gamma-viewer?label=gamma-viewer&amp;style=social" alt="AnswerCoalesce"></a></p><ul><li>ReasonerStdAPI Message Jupyter Notebook visualizer: <a href="https://github.com/ranking-agent/gamma-viewer" target="_blank" rel="noopener noreferrer">https://github.com/ranking-agent/gamma-viewer</a></li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="arax"></a>ARAX<a class="hash-link" href="#arax" title="Direct link to heading">#</a></h3><p><a href="https://github.com/RTXteam/RTX" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/RTXteam/RTX?label=RTX&amp;style=social" alt="RTX"></a></p><p>Team Expander Agent: <em>A tool for enhancing query graphs</em>. ARAX exposes all graph reasoning capabilities within a domain specific language: <strong>ARAXi</strong>.
ARAX is a tool for querying, manipulating, filtering, learning on, and exploring biomedical knowledge graphs.</p><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-8"></a>Access<a class="hash-link" href="#access-8" title="Direct link to heading">#</a></h5><ul><li>ARAX<ul><li>Open API: <a href="https://tiny.cc/arax-api-docs" target="_blank" rel="noopener noreferrer">https://tiny.cc/arax-api-docs</a></li><li>GitHub: <a href="https://github.com/RTXteam/RTX/tree/master/code/ARAX" target="_blank" rel="noopener noreferrer">https://github.com/RTXteam/RTX/tree/master/code/ARAX</a></li><li>GUI: <a href="https://arax.rtx.ai/" target="_blank" rel="noopener noreferrer">https://arax.rtx.ai/</a> </li><li><a href="https://github.com/RTXteam/RTX/blob/demo/code/ARAX/Examples/ARAX_Example3.ipynb" target="_blank" rel="noopener noreferrer">ARAX examples Notebook</a></li><li><a href="https://github.com/RTXteam/RTX/blob/master/code/ARAX/Documentation/DSL_Documentation.md" target="_blank" rel="noopener noreferrer">ARAXi documentation</a></li><li><a href="https://github.com/RTXteam/RTX/blob/master/README.md" target="_blank" rel="noopener noreferrer">Team/Tool/Code Readme</a></li></ul></li><li>RTX-KG2<ul><li>RTX-KG2 Neo4j UI: <a href="http://kg2canonicalized2.rtx.ai:7474" target="_blank" rel="noopener noreferrer">http://kg2canonicalized2.rtx.ai:7474</a></li><li>RTX-KG2 API: <a href="https://smart-api.info/ui/00bab7d59abe031098d5cb1597f7f1c4" target="_blank" rel="noopener noreferrer">https://smart-api.info/ui/00bab7d59abe031098d5cb1597f7f1c4</a></li><li>GitHub: <a href="https://github.com/RTXteam/RTX-KG2" target="_blank" rel="noopener noreferrer">https://github.com/RTXteam/RTX-KG2</a></li></ul></li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="explanatory-ara"></a>Explanatory ARA<a class="hash-link" href="#explanatory-ara" title="Direct link to heading">#</a></h3><ul><li><p>Analogical reasoning engine</p></li><li><p>Ranking results through explanations</p><ul><li>Explanatory evidence via NLU model</li><li>Explanatory evidence via other methods</li><li>Explaining information vs. explaining decisions</li></ul></li><li><p>Visualization of biomedical context</p></li><li><p>Open-world learning</p></li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="medikanren-20"></a>mediKanren 2.0<a class="hash-link" href="#medikanren-20" title="Direct link to heading">#</a></h3><p><a href="https://github.com/webyrd/mediKanren" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/webyrd/mediKanren?label=mediKanren&amp;style=social" alt="mediKanren"></a></p><p>Based on the <a href="http://minikanren.org/" target="_blank" rel="noopener noreferrer">miniKanren</a> logic programming language for reasoning over Knowledge Graoph (SemMedDB).</p><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-9"></a>Access<a class="hash-link" href="#access-9" title="Direct link to heading">#</a></h5><ul><li><a href="https://github.com/webyrd/mediKanren/tree/master/biolink" target="_blank" rel="noopener noreferrer">MediKanren BioLink interface on GitHub</a></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="deploy-4"></a>Deploy<a class="hash-link" href="#deploy-4" title="Direct link to heading">#</a></h5><ul><li>See <a href="https://github.com/webyrd/mediKanren#setup" target="_blank" rel="noopener noreferrer">documentation on GitHub</a>.</li></ul><hr><h3><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="improving-agent"></a>(im)proving agent<a class="hash-link" href="#improving-agent" title="Direct link to heading">#</a></h3><p><a href="https://github.com/baranzini-lab/PSEV" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/baranzini-lab/PSEV?label=PSEV&amp;style=social" alt="PSEV"></a></p><ul><li>SPOKE: a biomedical knowledge metagraph (~25 sources)<ul><li>reasoning to support facts from empirical evidences (Electronic Health Records, multi-omics studies)</li></ul></li></ul><p><a href="https://github.com/brettasmi/evidARA" target="_blank" rel="noopener noreferrer"><img src="https://img.shields.io/github/stars/brettasmi/evidARA?label=evidARA&amp;style=social" alt="evidARA"></a></p><ul><li>EvidARA<ul><li>takes query from ARS and extracts a graph q (output graph) from its internal Knowledge Network (SPOKE)</li><li>checks empirical evidence from raw data of cohorts (EHR and multi-omics studies)</li></ul></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="access-10"></a>Access<a class="hash-link" href="#access-10" title="Direct link to heading">#</a></h5><ul><li>SPOKE<ul><li>GitHub: <a href="https://github.com/baranzini-lab/PSEV" target="_blank" rel="noopener noreferrer">https://github.com/baranzini-lab/PSEV</a></li></ul></li><li>evidARA<ul><li>GitHub: <a href="https://github.com/brettasmi/evidARA" target="_blank" rel="noopener noreferrer">https://github.com/brettasmi/evidARA</a></li></ul></li></ul><h5><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="deploy-5"></a>Deploy<a class="hash-link" href="#deploy-5" title="Direct link to heading">#</a></h5><ul><li>SPOKE on neo4j. See <a href="https://github.com/baranzini-lab/PSEV" target="_blank" rel="noopener noreferrer">documentation on GitHub</a>.</li></ul><hr><h2><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="ars"></a>ARS<a class="hash-link" href="#ars" title="Direct link to heading">#</a></h2><p>ARS registry: <a href="https://ars.transltr.io/ars/app/status" target="_blank" rel="noopener noreferrer">https://ars.transltr.io/ars/app/status</a></p><hr><h2><a aria-hidden="true" tabindex="-1" class="anchor enhancedAnchor_prK2" id="additional-translator-resources"></a>Additional Translator resources<a class="hash-link" href="#additional-translator-resources" title="Direct link to heading">#</a></h2><p><a href="https://github.com/vemonet/translator-sparql-notebook/blob/master/translator-relay-cluster4.ipynb" target="_blank" rel="noopener noreferrer">Jupyter Notebook to combine data from various Knowledge Providers</a>, produced during the Relay Days.</p></div></article><div class="margin-vert--xl"><div class="row"><div class="col"><a href="https://github.com/MaastrichtU-IDS/d2s-docs/edit/master/website/docs/translator-prototypes-registry.md" target="_blank" rel="noreferrer noopener"><svg fill="currentColor" height="1.2em" width="1.2em" preserveAspectRatio="xMidYMid meet" role="img" viewBox="0 0 40 40" class="iconEdit_2LL7"><g><path d="m34.5 11.7l-3 3.1-6.3-6.3 3.1-3q0.5-0.5 1.2-0.5t1.1 0.5l3.9 3.9q0.5 0.4 0.5 1.1t-0.5 1.2z m-29.5 17.1l18.4-18.5 6.3 6.3-18.4 18.4h-6.3v-6.2z"></path></g></svg>Edit this page</a></div><div class="col text--right"><em><small>Last updated on <time datetime="2022-06-17T10:30:51.000Z" class="docLastUpdatedAt_1Qna">6/17/2022</time> by <strong>Vincent Emonet</strong></small></em></div></div></div><div class="margin-vert--lg"><nav class="pagination-nav" aria-label="Blog list page navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/docs/deploy-additional-services"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« Additional services</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/docs/guide-preprocessing"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Preprocess input files Â»</div></a></div></nav></div></div></div><div class="col col--3"><div class="tableOfContents_2xL- thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#standard-reasoner-implementations-sri" class="table-of-contents__link">Standard Reasoner Implementations (SRI)</a><ul><li><a href="#reasonerapi" class="table-of-contents__link">ReasonerAPI</a></li><li><a href="#kgx" class="table-of-contents__link">KGX</a></li><li><a href="#nodenormalization-service" class="table-of-contents__link">NodeNormalization Service</a></li><li><a href="#edgenormalization-service" class="table-of-contents__link">EdgeNormalization Service</a></li><li><a href="#nameresolution" class="table-of-contents__link">NameResolution</a></li><li><a href="#biolink-model-lookup-service" class="table-of-contents__link">BioLink Model Lookup service</a></li><li><a href="#blcompliance" class="table-of-contents__link">BLCompliance</a></li></ul></li><li><a href="#knowledge-providers-kp" class="table-of-contents__link">Knowledge Providers (KP)</a><ul><li><a href="#cohd-clinical-data2services-provider" class="table-of-contents__link">COHD Clinical Data2Services Provider</a></li><li><a href="#molepro-molecular-data-provider" class="table-of-contents__link">MolePro Molecular Data Provider</a></li><li><a href="#genetic-knowledge-provider" class="table-of-contents__link">Genetic Knowledge Provider</a></li><li><a href="#icees-kp-exposure-provider" class="table-of-contents__link">ICEES+ KP Exposure Provider</a></li><li><a href="#text-mining-provider" class="table-of-contents__link">Text Mining Provider</a></li><li><a href="#connections-hypothesis-provider" class="table-of-contents__link">Connections Hypothesis Provider</a></li><li><a href="#docket-multiomics-provider" class="table-of-contents__link">DOCKET multiomics provider</a></li><li><a href="#biothings-api" class="table-of-contents__link">BioThings API</a></li></ul></li><li><a href="#autonomous-relay-agent-ara" class="table-of-contents__link">Autonomous Relay Agent (ARA)</a><ul><li><a href="#biothings-explorer" class="table-of-contents__link">BioThings Explorer</a></li><li><a href="#aragorn" class="table-of-contents__link">ARAGORN</a></li><li><a href="#arax" class="table-of-contents__link">ARAX</a></li><li><a href="#explanatory-ara" class="table-of-contents__link">Explanatory ARA</a></li><li><a href="#medikanren-20" class="table-of-contents__link">mediKanren 2.0</a></li><li><a href="#improving-agent" class="table-of-contents__link">(im)proving agent</a></li></ul></li><li><a href="#ars" class="table-of-contents__link">ARS</a></li><li><a href="#additional-translator-resources" class="table-of-contents__link">Additional Translator resources</a></li></ul></div></div></div></div></main></div></div><footer class="footer"><div class="container"><div class="text--center"><div><a rel="noopener noreferrer" href="https://creativecommons.org/licenses/by/4.0/" title="Creative Commons Attribution 4.0 International license" target="_blank"><img src="/img/cc-by.svg" alt="cc by license"></a></div><div style="margin-bottom:10px">Copyright Â© 2021 Institute of Data Science at Maastricht University</div><div style="margin-bottom:10px;color:grey">We track page views and users demographics using Google Analytics to improve our users experience. <a href="https://policies.google.com/technologies/partner-sites">See how Google uses collected informations</a>.<br>You can prevent Google Analytics tracking by enabling <a href="https://blog.mozilla.org/blog/2019/06/04/firefox-now-available-with-enhanced-tracking-protection-by-default/">Firefox Tracking Protection</a>, installing <a href="https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=fr">uBlock Origin add-on</a>, or using the official <a href="https://tools.google.com/dlpage/gaoptout/">Google Analytics Opt-out add-on</a>.</div></div></div></footer></div>
<script src="/styles.903fb0a7.js"></script>
<script src="/runtime~main.7c9aaa28.js"></script>
<script src="/main.c3417a16.js"></script>
<script src="/1.7b322362.js"></script>
<script src="/2.ab994efe.js"></script>
<script src="/1f391b9e.64b390a9.js"></script>
<script src="/70.e1de10fa.js"></script>
<script src="/935f2afb.8c78b57b.js"></script>
<script src="/17896441.2fd276c4.js"></script>
<script src="/6a8fabba.e3f4939a.js"></script>
</body>
</html>